packages/connector/src/utils.ts

```diff
@@ -51,3 +53,116 @@ export async function validateAndSetCeramicSession(
     throw new Error('Failed to set session in Masca.');
   }
 }
+
+export async function signVP(presentation: VerifiablePresentation) {
```

Comment:
I would rename both methods so we write out their full names -> `signVerifiableCredential`, `signVerifiablePresentation` ̇